lib/ansible/module_utils/net_tools/nios/api.py

```diff
@@ -50,7 +50,8 @@
     'http_pool_connections': dict(type='int', default=10),
     'http_pool_maxsize': dict(type='int', default=10),
     'max_retries': dict(type='int', default=3),
-    'wapi_version': dict(default='1.4')
+    'wapi_version': dict(default='1.4'),
+    'max_results': dict(type='int', default=1000)
```

Comment:
Please could you document this here: https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/nios.py

Reply:
Hello @gundalow 

Of course, I'll do it when I'm at home.
I do not have what it takes in my workplace to be able to do it.

Reply:
Hello @gundalow

Done.

Best regards,